bounties/npm/node-lsof/1/vulnerability.json

```diff
@@ -0,0 +1,50 @@
+{
+	"PackageVulnerabilityID": 1,
+	"DisclosureDate": "2020-08-17",
+	"AffectedVersionRange": "*",
+	"Summary": "Command Injection",
+	"Contributor": {
+		"Discloser": "",
+		"Fixer": ""
+	},
+	"Package": {
+		"Registry": "npm",
+		"Name": "lsof",
+		"URL": "https://www.npmjs.com/package/lsof",
+		"Downloads": "247506"
+	},
+	"CWEs": [
+		{
+			"ID": "CWE-78",
+			"Description": ""
+		}
+	],
+	"CVSS": {
+		"Version": "3.1",
+		"AV": "N",
+		"AC": "L",
+		"PR": "N",
+		"UI": "N",
+		"S": "U",
+		"C": "H",
+		"I": "N",
+		"A": "N",
+		"E": "",
+		"RL": "",
+		"RC": "",
+		"Score": "7.5"
+	},
+	"CVEs": [
+		"CVE-2019-10783"
+	],
+	"Repository": {
+		"URL": "https://github.com/davglass/node-lsof",
+		"Codebase": [
+			"JavaScript"
+		]
+	},
+	"Permalinks": [
```

Comment:
Please add also other permalinks:

* https://github.com/davglass/node-lsof/blob/master/lib/lsof.js#L9
* https://github.com/davglass/node-lsof/blob/master/lib/lsof.js#L71

Cheers,
Mik

Reply:
Suggestions added, any further info/references on bounties that you can see as additions are always a great help.

Cheers ❤️